src/core/OuterExamTimer.jsx

```diff
@@ -1,23 +1,32 @@
 import React, { useEffect, useContext } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import { AppContext } from '@edx/frontend-platform/react';
-import ExamStateContext from '../context';
 import { ExamTimerBlock } from '../timer';
 import ExamAPIError from '../exam/ExamAPIError';
-import ExamStateProvider from './ExamStateProvider';
+import {
+  getLatestAttemptData,
+  stopExam,
+  submitExam,
+  expireExam,
+  pollAttempt,
+  pingAttempt,
+} from '../data/thunks';
+import { IS_STARTED_STATUS } from '../constants';
 
 const ExamTimer = ({ courseId }) => {
-  const state = useContext(ExamStateContext);
   const { authenticatedUser } = useContext(AppContext);
+
   const {
-    activeAttempt, showTimer, stopExam, submitExam,
-    expireExam, pollAttempt, apiErrorMsg, pingAttempt,
-    getLatestAttemptData,
-  } = state;
+    activeAttempt, apiErrorMsg,
+  } = useSelector(state => state.specialExams);
+  const dispatch = useDispatch();
+
+  const showTimer = !!(activeAttempt && IS_STARTED_STATUS(activeAttempt.attempt_status));
```

Comment:
TIL what !! is - cooool! Love "bang, bang you're boolean" here: https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript
Nice!